pkg/assetfetcher/asset.go

```diff
@@ -330,7 +333,7 @@ func (a Asset) AssetPayloadToAssetEvent() (domain.AssetEvent, error) {
 type assetHistoryEvents []AssetHistory
 
 func (a assetHistoryEvents) lastScannedTimestamp() (time.Time, error) {
-	latestTime := time.Time{}
+	latestTime := time.Time{} // empty struct
```

Comment:
nit: the typical go form of this is `var latestTime time.Time` which conveys "a variable with the zero value of type X"